### ISSUE TITLE
I cannot login to CP as admin

### ISSUE THREAD

**Issue by rmalenko** [serghey-rodin/vesta#312]
I cannot login to CP as admin. I have tried changing password from web form, server console for user admin. However, I still cannot login.
Could you prompt me what I can do or where need to check?

**Comment by crackdepirate** [comment]
Have you tried with "root" as username ?

**Comment by rmalenko** [comment]
Yes, I tried.

**Comment by rmalenko** [comment]
Thank you for answer
```
2015-02-11 13:21:57 admin 188.31.53.6 successfully logged in 
2015-02-11 13:42:56 root 188.31.53.6 successfully logged in
```
```
2015/02/11 14:09:42 [error] 19978#0: *53 FastCGI sent in stderr: "PHP message: PHP Warning:  Invalid argument supplied for foreach() in /usr/local/vesta/web/login/index.php on line 33
PHP message: PHP Notice:  Undefined index: LANGUAGE in /usr/local/vesta/web/login/index.php on line 38" while reading response header from upstream, client: 188.31.53.6, server: _, request: "GET /login/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/vesta-php.sock:", host: "elindesign.cu:8083"
```

I.e. is something wrong with web interface?

**Comment by TheLastProject** [comment]
Duplicate of #301?

Try removing empty lines on the end of vesta.conf if they exist, that may fix the issue.

**Comment by rmalenko** [comment]
I am not sure. I have one empty line in vesta.conf.

I am running Linux-box and have installed fresh Vesta, then I copied all files with replace into production server. Now I can login.

**Comment by joshbmarshall** [comment]
Try this: I log in as "admin" user and it has a blank config. So I edited the login file and found that it executes "sudo v-list-sys-config json" as the "admin" user.

So I executed that file as "admin" and found that I got this result:

[admin@joshtest ~]$ /usr/bin/sudo /usr/local/vesta/bin/v-list-sys-config json
wc: /conf/vesta.conf: No such file or directory
{
	"config": {
cat: /conf/vesta.conf: No such file or directory
	}
}

So I believe the profile isn't being passed through to sudo. Ensure your sudoers file has these lines:

Defaults    env_reset
Defaults    env_keep = "COLORS DISPLAY HOSTNAME HISTSIZE INPUTRC KDEDIR \
                        LS_COLORS MAIL PS1 PS2 QTDIR USERNAME \
                        LANG LC_ADDRESS LC_CTYPE LC_COLLATE LC_IDENTIFICATION \
                        LC_MEASUREMENT LC_MESSAGES LC_MONETARY LC_NAME LC_NUMERIC \
                        LC_PAPER LC_TELEPHONE LC_TIME LC_ALL LANGUAGE LINGUAS \
                        _XKB_CHARSET XAUTHORITY VESTA"

That got logging in working for me!
